src/System.Security.Cryptography.Cng/src/Interop/NCrypt/NCrypt.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
```

Comment:
Algorithms uses BCrypt calls, not NCrypt calls so there’s no overlap. The RSACng code might have more overlap – will keep that in mind. The Cng contract surfaces so many enums that map directly to NCRYPT constants that the interop layer is tied to the Cng contract anyway, unless we want a whole bunch of enum->int casts in the code.